analyzer/src/main/kotlin/managers/Pub.kt

```diff
@@ -520,3 +521,26 @@ class Pub(
         run(workingDir, "get")
     }
 }
+
+/**
+ * Extract information about package authors from the given [pubspec].
+ */
+private fun fetchAuthors(pubspec: JsonNode): SortedSet<String> {
```

Comment:
Consider naming this `getAuthors`. The term `fetch` to me indicates something little more heavy weight, like a network request or so. That's very subjective, it may be just me.

Consider turing this function into an extension of JsonNode to make the code a bit more compact.

Reply:
I agree it would make sense to align the function naming with the previously merged related NPM code.

Reply:
The function has been renamed to `parseAuthors()`, which is the name used by `Npm`.

I locally tried to convert it to an extension function, but did not like the result so much. The amount of code that could be reduced is rather low, and access to the properties of the node has to be written as `this["author"]`.